### ISSUE TITLE
CausalTestResult test_value

### ISSUE THREAD

**Issue by jmafoster1** [CITCOM-project/CausalTestingFramework#196]
We have a field in CausalTestResult called `test_value`. Please can someone tell me what it is? Have we always had it? Looking at how it's used, it seems like redundancy for `Estimator.treatment_value`. If this is the case, can it be removed, or do we use it for something special?

**Comment by jmafoster1** [comment]
Along similar lines, I really don't think we should be passing `Estimator` objects into `CausalTestResult` objects, especially since we're only accessing particular properties. I think it would be much better to put those values directly in fields, or if pylint kicks off, we should have a data class or something.

**Comment by cwild-UoS** [comment]
> We have a field in CausalTestResult called `test_value`. Please can someone tell me what it is? Have we always had it? Looking at how it's used, it seems like redundancy for `Estimator.treatment_value`. If this is the case, can it be removed, or do we use it for something special?

Yeah I see what you mean, it seems a bit redundant to both wrap the estimator value and pass it in, as well as passing in the estimator it self. It came in with PR #119 which separated `CausalTestResult` from `causal_test_outcome.py` as well as other refactoring.

It seems the `TestValue` data class also adds the value type as a string, which was not a thing before and is used for applying different type in the causal outcomes:

https://github.com/CITCOM-project/CausalTestingFramework/blob/d6da6a3adb50e057e6cc55f6d95b074b49206cdc/causal_testing/testing/causal_test_outcome.py#L87
